### DISCUSSION TITLE
Add "Always show all tray icons" from Schneegans and add a scheduled task to rerun

### DISCUSSION THREAD

**Discussion by rhasselbusch** [memstechtips/UnattendedWinstall#119] [Ideas]
The [Schneegans XML Generator](https://schneegans.de/windows/unattend-generator/) uses a Powershell script to enumerate through the Tray Notification Items and set their promoted value to 1. If we could implement the same and create a scheduled task to rerun the script on system startup for example, we always have all TNA icons visible.

The disadvantage of this solution would be that you can't hide icons you really won't show up.

**Comment by memstechtips** [comment]
Could you post a screenshot or paste the lines here that the schneegans generator uses?

**Comment by memstechtips** [comment]
Hey there and sorry for the late reply,

Since the creation of this issue I have released a new version of the XML file and a major update to [Winhance](https://winhance.net/), I recommend that you watch [this video](https://youtu.be/lrq3ph3xi50) which provides a full demo of the current version along with how to use the new version of WIMUtil and how to install the drivers etc. You now also have the ability to create an autounattend.xml file based on your selections in Winhance, so you have more control over which apps are removed and which settings are applied.

I have also created [this video](https://youtu.be/sCd1TI6AbOI) about v3.0.0 of UnattendedWinstall, I suggest that you watch it too if you're interested.

This is enabled by default and you can also toggle it in Winhance.

I'm closing this issue now, but if you need more help with the same issue, feel free to reopen it.
